### PR TITLE
Add format_policies to pretty-print Cedar policies

### LINKED ISSUE #4: Add function to format policies
Add a function to format policies using `cedar_policy_formatter::policies_str_to_pretty`.  See [cli](https://github.com/cedar-policy/cedar/blob/217d8c31df6a6eda95bfc218d41c9176a3379f88/cedar-policy-cli/src/lib.rs#L435) for an approach.

### PR DESCRIPTION
**Pull request by glb**
Based on the [CLI](https://github.com/cedar-policy/cedar/blob/217d8c31df6a6eda95bfc218d41c9176a3379f88/cedar-policy-cli/src/lib.rs#L435) implementation.

Fixes #4.

### CODE REVIEW

**Comment by glb** [comment]
Well _this_ is neat: looks like the files are there but for some reason `pip` is downloading anyway.

[logs](https://github.com/k9securityio/cedar-py/actions/runs/5486784114/jobs/9997333084?pr=12#step:6:22):

```
# ls dist
cedarpy-0.1.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
cedarpy-0.1.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
cedarpy-0.1.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
cedarpy-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
cedarpy-0.1.0-pp310-pypy310_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
cedarpy-0.1.0-pp39-pypy39_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
# pip install cedarpy --find-links dist --force-reinstall
Looking in links: dist
Collecting cedarpy
  Downloading cedarpy-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.9/2.9 MB 11.8 MB/s eta 0:00:00
Installing collected packages: cedarpy
Successfully installed cedarpy-0.1.0
```

I wonder why `pip` is downloading the file when it's clearly present? 🤔

**Comment by glb** [comment]
So it seems that the local file is indeed being found:

[log](https://github.com/k9securityio/cedar-py/actions/runs/5486977263/jobs/9997785601?pr=12#step:6:864)

```
  Found link https://files.pythonhosted.org/packages/27/4d/a3d6d9edb8224ce09abfe578d3c9faa93bb20c265b00edfaba59e437e74a/cedarpy-0.1.0.tar.gz (from https://pypi.org/simple/cedarpy/) (requires-python:>=3.9), version: 0.1.0
```

[another log](https://github.com/k9securityio/cedar-py/actions/runs/5486977263/jobs/9997785601?pr=12#step:6:868)

```
Local files found: /home/runner/work/cedar-py/cedar-py/dist/cedarpy-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

However, there are multiple matches found:

```text
  Found link https://files.pythonhosted.org/packages/1d/d4/1f18450d56579632df3f58dd66c61278e12e895e0e088fc1721fa7a9c90e/cedarpy-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (from https://pypi.org/simple/cedarpy/) (requires-python:>=3.9), version: 0.1.0
  Found link https://files.pythonhosted.org/packages/27/4d/a3d6d9edb8224ce09abfe578d3c9faa93bb20c265b00edfaba59e437e74a/cedarpy-0.1.0.tar.gz (from https://pypi.org/simple/cedarpy/) (requires-python:>=3.9), version: 0.1.0
Found link file:///home/runner/work/cedar-py/cedar-py/dist/cedarpy-0.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl, version: 0.1.0
```

and from the [docs](https://pip.pypa.io/en/latest/cli/pip_install/#finding-packages):

> pip looks for packages in a number of places: on PyPI (if not disabled via --no-index), in the local filesystem, and in any additional repositories specified via --find-links or --index-url. There is no ordering in the locations that are searched. Rather they are all checked, and the “best” match for the requirements (in terms of version number - see [PEP 440](https://peps.python.org/pep-0440/) for details) is selected.

so I'm guessing that the problem is that `pip` is "randomly" (but consistently) picking the files from PyPI instead of the local one.

Trying `--no-index` on the `cedarpy` install to see if disabling PyPI solves the issue.

**Comment by skuenzli** [comment]
The `format_policies` implementation looks great! Thank you!

File: tests/unit/test_format_policy.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from textwrap import dedent
```
Comment (skuenzli) [inline tests/unit/test_format_policy.py:3]: TIL about the `textwrap` package.  Very cool.

**Comment by skuenzli** [comment]
Some projects featured by maturin as examples of use have also adopted `--no-index`
* [rjmespath-py](https://github.com/messense/rjmespath-py/blob/main/.github/workflows/CI.yml)
* [pydantic-core](https://github.com/pydantic/pydantic-core/blob/main/.github/workflows/ci.yml#L477)

A number of featured projects have also rewritten their CI.yml pretty substantially.

At any rate `pip install --help` confirms we should use `-no-index` our situation:
```
  --no-index                  Ignore package index (only looking at --find-links URLs instead).
```

**Pull request merged**